roles/common/tasks/main.yml

```diff
@@ -0,0 +1,8 @@
+---
+- name: "Verify pod - {{ common_pod_test_id }}"
+  when:
+    - common_pod_list is defined
+    - common_pod_nspace is defined
+    - common_pod_status_str is defined
+  ansible.builtin.include_tasks: "pod_tests.yml"
+  loop: "{{ common_pod_list }}"
```

Comment:
This should include a similar pattern for the CRD tests, where the tests are run conditionally, based on the vars existing